fabric-rendering-v1/src/main/java/net/fabricmc/fabric/mixin/client/rendering/v1/MixinBlockColorMap.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package net.fabricmc.fabric.mixin.client.rendering;
+package net.fabricmc.fabric.mixin.client.rendering.v1;
```

Comment:
mixin and impl packages don't carry the version, it is only meant to differentiate the API classes - mods aren't supposed to access impl classes, voiding the need for multiple copies.